docs/docs/workflows/terraform-module.md

```diff
@@ -0,0 +1,151 @@
+# Use the Terraform module
+You can manage a Constellation cluster through Terraform.
+The module package is available as part of the [GitHub release](https://github.com/edgelesssys/constellation/releases/). It consists of a convenience module for each cloud service provider (`{csp}-constellation`) that combines the IAM (`infrastructure/{csp}/iam`), infrastructure (`infrastructure/{csp}`), and constellation (`constellation-cluster`) modules.
+
+## Prerequisites
+- a Linux / Mac operating system
+- a Terraform installation of version `v1.4.4` or above
+
+## Quick setup
+The convenience module allows setting up a Constellation cluster with a single module. It's easiest to consume the module through a remote source, as shown below.
+This allows to upgrade the cluster to a newer Constellation version by simply updating the module source.
+
+:::caution
+
+In the current release of the module, `terraform apply` creates files such as `constellation-conf.yaml`, `constellation-state.yaml` , `constellation-admin.conf`, `constellation-mastersecret.json`, and a directory `constellation-terraform"` containing backups. Make sure to check in these files in your version control when using GitOps.
+The files are deleted on `terraform destroy`.
+
+:::
+
+1. Create a directory (workspace) for your Constellation cluster.
+  ```bash
+  mkdir constellation-workspace
+  cd constellation-workspace
+  ```
+
+1. Create a `main.tf` file to call the CSP specific Constellation module.
+
+  <tabs groupId="csp">
+
+  <tabItem value="azure" label="Azure">
+
+  ```
+  module "azure-constellation" {
+    source = "https://github.com/edgelesssys/constellation/releases/download/<version>/terraform-module.zip//terraform-module/azure-constellation" // insert Constellation version here.
```

Comment:
```suggestion
    // replace <version> with a Constellation version, e.g., v2.13.0
    source = "https://github.com/edgelesssys/constellation/releases/download/<version>/terraform-module.zip//terraform-module/azure-constellation"
```
* The comment at the end of the line can easily be overlooked.
* We should give a hint on the format of the version number.